src/managers/admin-api.ts

```diff
@@ -68,6 +68,18 @@ export class AdminManager implements AdminApi {
     return deleteResult;
   }
 
+  public async approveElevatorBooking(id: number, formData: FormData): Promise<boolean> {
```

Comment:
Should admins also have the ability to disapprove bookings?

Reply:
I'm going to say no for now. The next step is some automated emails. If they approve and then change their mind they will need to reach out anyway.